tests/bootstrap.php

```diff
@@ -88,7 +88,7 @@
 
 // Ensure default test connection is defined
 if (!getenv('db_dsn')) {
-    putenv('db_dsn=sqlite://' . TMP . 'debug_kit_test.sqlite');
+    putenv('db_dsn=sqlite:///' . TMP . 'debug_kit_test.sqlite');
```

Comment:
I used 3 like in the tests: https://github.com/cakephp/cakephp/blob/master/tests/TestCase/Core/StaticConfigTraitTest.php#L222